client/js/views/GameView.vue

```diff
@@ -1108,7 +1108,9 @@ export default {
   color: #fff;
   width: 100vw;
   height: 100%;
-  background: linear-gradient(180deg, #6202ee 14.61%, #fd6222 100%), #c4c4c4;
+  background-image: url('../img/game_images/board_background.svg');
```

Comment:
Do we have aliases set up to use instead of these relative paths?

Reply:
My webpack chops are weak, but if anything it would be 


```suggestion
  background-image: url('@/js/img/game_images/board_background.svg');
```
right? @peterancourt wanna give that a try?

Reply:
You can leave it for now, I redid a lot of this stuff in the Vue3 branch which we'll have to change anyway 🤷🏼

Reply:
It looks like it would be `@/img/game_images/board_background.svg`, but with that the CSS is trying to load `./@/img/game_images/board_background.svg` - does the webpack naming convention only work for JS?

Reply:
Ah it was `~@/img/game_images/board_background.svg`, the more you know